specs/language/basic.tex

```diff
@@ -1,5 +1,97 @@
 \Ch{Basic Concepts}{Basic}
 
+\Sec{Types}{Basic.types}
+
+\p The \textit{object representation} of an object of type \texttt{T} is the
+sequence of \textit{N} bytes taken up by the object of type \texttt{T}, where
+\textit{N} equals \texttt{sizeof(T)}\footnote{\texttt{sizeof(T)}, returns the
+size of an object of type \texttt{T} in bytes as stored in device memory.
+The size may be different in other memory spaces and no operator exists to query
+it.}. The \textit{object representation} of an object may be different based on
+the \textit{memory space} it is stored in (\ref{Intro.Memory.Spaces}).
+
+\p The \textit{value representation} of an object is the set of bits that hold
+the value of type \texttt{T}.
```

Comment:
Should we mention padding? That is, the "object representation" presumably includes padding, and the "value representation" presumably does not, as in C++.